DataFormats/HcalDigi/test/BuildFile.xml

```diff
@@ -1,7 +1,13 @@
-<use   name="boost"/>
-<use   name="DataFormats/HcalDigi"/>
-<use   name="FWCore/Framework"/>
-<use   name="FWCore/ParameterSet"/>
 <library   file="HcalDigiDump.cc" name="HcalDigiDump">
   <flags   EDM_PLUGIN="1"/>
+  <use   name="boost"/>
+  <use   name="DataFormats/HcalDigi"/>
+  <use   name="FWCore/Framework"/>
+  <use   name="FWCore/ParameterSet"/>
 </library>
+<bin name="test_hcal_digi" file="test_hcal_digi.cu">
```

Comment:
@vkhristenko , please add these tests for only AMD64. Cuda is not available on aarch64 and this breaks aarch64 IBs/releases. You can do something like this https://github.com/cms-sw/cmssw/blob/master/DataFormats/DetId/test/BuildFile.xml to only build/run cuda based tests for amd64.

Reply:
thanks a lot for the note! will do like that from now on...